curvefs/conf/mds.conf

```diff
@@ -94,3 +94,43 @@ mds.scheduler.metaserver.cooling.timeSec=1800
 # the backend thread check whether fs is able to delete,
 # check partition of deleting fs is deleting
 mds.fsmanager.backEndThreadRunInterSec=10
+
+#### s3
+# the max size that fuse send
+s3.fuseMaxSize=131072
+s3.pagesize=65536
+# prefetch blocks that disk cache use
+s3.prefetchBlocks=1
+# prefetch threads
+s3.prefetchExecQueueNum=1
+# start sleep when mem cache use ratio is greater than nearfullRatio,
+# sleep time increase follow with mem cache use raito, baseSleepUs is baseline.
+s3.nearfullRatio=70
+s3.baseSleepUs=500
```

Comment:
these fields are useless.

Reply:
> these fields are useless.

this for [init](https://github.com/opencurve/curve/blob/6c0a820be0f5f7b232913c91bc326369bb78f091/curvefs/src/mds/mds.cpp#L159) but i will try for delete this